common/jenkins-agents/terraform/docker/Gemfile

```diff
@@ -0,0 +1,19 @@
+source 'https://rubygems.org' do
```

Comment:
Few weeks ago I tested such a mirror using `bundle config --local mirror.https://rubygems.org http://odsnexus/repository/path`. Prerequisite is a proper repo setup. While testing nexus was slower than "native internet speed".
Is there a preset variable for it available, should it be an additional ENV settings or just a configuration option in case defined within the ODS project? For this project it would make sense to also provide a mirror for packagecloud.io.